src/sqlfluff/dialects/dialect_bigquery_keywords.py

```diff
@@ -58,7 +58,6 @@
 LIMIT
 LOOKUP
 MERGE
-NATURAL
```

Comment:
This was down as an unreserved word in BigQuery (I think I copied this from ANSI when setting up the BigQuery word list) but is not needed. Test case added to ensure it doesn't fail in future.